cafebabel/templates/articles/detail.html

```diff
@@ -105,7 +105,14 @@ <h1>{{ article.title }}</h1>
   {% block share_nav %}
   <nav class=share>
   <p>Loved this story? Then tell your friends:
-    <a href="https://twitter.com/share?url={{ request.url|quote_plus }}&text={{ article.title }}&via=cafebabel_eng">Twitter</a>
+    {% set lang = current_language %}
+    {% if lang == 'en' %}{% set twitter_nickname = 'cafebabel_ENG' %}{% endif %}
```

Comment:
Can you keep "eng" lowercase as that breaks the tests?
If not the test need to be adapted to have "ENG" uppercase ;)

Reply:
ha ! sure ! I thought we need to keep the right case ! good https://github.com/cafebabel/cafebabel.com/pull/486/commits/6659587ebae852f988c107fbd6be6c1ffbc56c9c